build.sh

```diff
@@ -145,6 +147,10 @@ cmdline() {
           *windows-10-enterprise*)
             export ISO_URL="https://software-download.microsoft.com/download/pr/19041.264.200511-0456.vb_release_svc_refresh_CLIENTENTERPRISEEVAL_OEMRET_x64FRE_en-us.iso"
           ;;
+          *windows-10-ltsc*)
+            WINDOWS_VERSION="10-ltsc"
```

Comment:
This will not work with `guest_os_type` inside `windows.json`:
https://github.com/ruzickap/packer-templates/blob/1aca4ec6e4787ac4aba9995242c347210e54f835/windows.json#L51

If you change the `WINDOWS_VERSION="10-ltsc"` then it will not work with VirtualBox. The VirtualBox will look for `Windows10-ltsc_64` which does not exists:

```bash
$ VBoxManage list ostypes | grep "^ID:.*Windows10"
ID:          Windows10
ID:          Windows10_64
```